wrappercommon/pom.xml

```diff
@@ -35,9 +35,15 @@
 			<version>2.11.0</version>
 		</dependency>
 		<dependency>
-			<groupId>org.apache.ws.security</groupId>
-			<artifactId>wss4j</artifactId>
-			<version>1.6.19</version>			
+			<groupId>org.apache.wss4j</groupId>
```

Comment:
Why we change wss4j version?
The only thing that we need to add is de exclusion.